handlers/sf-billing-account-remover/src/main/scala/com/gu/sf_billing_account_remover/BillingAccountRemover.scala

```diff
@@ -172,7 +168,7 @@ object BillingAccountRemover extends App with LazyLogging {
   }
 
   def doSfGetWithQuery(sfAuthDetails: SfAuthDetails, query: String): String = {
-    Http(s"${sfAuthDetails.instance_url}/services/data/v$salesforceApiVersion/query/")
+    Http(s"${sfAuthDetails.instance_url}/services/data/v54.0/query/")
```

Comment:
Would this not fix it?

```suggestion
    Http(s"${sfAuthDetails.instance_url}/services/data/v${salesforceApiVersion}/query/")
```

Reply:
If not, using v54 is fine for the next few years :)

Reply:
I think you're right! I've readded the variable and usage with the correct syntax. Nice spot

Reply:
Does  this work? I can't see why it would as `$salesforceApiVersion` is valid syntax, you only need the curly braces when the thing you're injecting is multipart eg `${user.name}`

Reply:
Rupert is correct, it didn't work. The syntax was correct all along. I've reverted it now so that the api version is hardcoded